salt/orch/kubernetes.sls

```diff
@@ -35,15 +35,24 @@ set-etcd-roles:
       - set-bootstrap-in-progress-flag
 {% endif %}
 
-sync-pillar:
-  salt.runner:
-    - name: saltutil.sync_pillar
+update-modules:
+  salt.function:
+    - tgt: '*'
+    - name: saltutil.sync_all
+    - kwarg:
+        refresh: True
     - require:
       - set-bootstrap-in-progress-flag
 {%- if additional_etcd_members|length > 0 %}
       - set-etcd-roles
 {%- endif %}
 
+sync-pillar:
+  salt.runner:
+    - name: saltutil.sync_pillar
+    - require:
+      - update-modules
+
 update-pillar:
```

Comment:
On logic this should be before synchronizing pillars. First we are updating value and then we are synchronizing it to all. Is it right?

Reply:
I'm not sure I understand the question?

Reply:
I asked about order 
```
sync-pillar:
update-pillar:
```
I have checked documentation https://docs.saltstack.com/en/latest/ref/modules/all/salt.modules.saltutil.html and looks OK for now.